specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json

```diff
@@ -367,6 +367,10 @@
           "items": {
             "$ref": "#/definitions/MetadataDTO"
           }
+        },
+        "strictFiltersCompoundOperationType": {
```

Comment:
also, make the change v5 preview too

Reply:
changes made to v5 preview too.

---
In reply to: [454802801](https://github.com/Azure/azure-rest-api-specs/pull/10132#discussion_r454802801) [](ancestors = 454802801)